legacy/firmware/layout2.c

```diff
@@ -586,7 +586,9 @@ void layoutConfirmModifyOutput(const CoinInfo *coin, AmountUnit amount_unit,
 }
 
 void layoutConfirmModifyFee(const CoinInfo *coin, AmountUnit amount_unit,
-                            uint64_t fee_old, uint64_t fee_new) {
+                            uint64_t fee_old, uint64_t fee_new,
+                            uint64_t total_in, uint64_t total_out,
```

Comment:
`fee_new == total_in - total_out`, no need to pass these two

Reply:
Good catch, it simplifies stuff a bit. Done in [3cfe21d](https://github.com/trezor/trezor-firmware/pull/2481/commits/3cfe21dd18626d57d1326b9f9ca1c79f0f88e33a)